commitlint/helpers.ts

```diff
@@ -37,6 +37,14 @@ export abstract class Helpers {
         return text as string;
     }
 
+    public static assertWhen(when: string) {
+        if (when !== "never" && when !== "always") {
```

Comment:
@realmarv I've realised we should create an enum called "When" that has both When.Never and When.Always (but instead of mapping to int, they map to string); hopefully typescript supports this?